source/renderer/assets/context/tabs.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "label": "menu.rename_file",
+    "command": "file-rename"
+  },
+  {
+    "label": "menu.delete_file",
+    "command": "file-delete"
+  },
+  {
+    "label": "menu.duplicate_file",
+    "command": "file-duplicate"
+  },
+  {
+    "type": "separator"
+  },
+  {
+    "label": "menu.close_all",
```

Comment:
Please rename to, e.g., `menu.close_all_tabs`, because otherwise we would run into problems with regard to windows, and notify me of your chosen identifier, so that I can add it to Zettlr Translate!

Reply:
Done. Went with your suggestion of `menu.close_all_tabs`.